src/html/htmprint.cpp

```diff
@@ -709,8 +709,11 @@ bool wxHtmlEasyPrinting::DoPreview(wxHtmlPrintout *printout1, wxHtmlPrintout *pr
     }
 
     wxPreviewFrame *frame = new wxPreviewFrame(preview, m_ParentWindow,
-                                               m_Name + _(" Preview"),
-                                               wxPoint(100, 100), wxSize(650, 500));
+                                               wxString::Format(_("%s Preview"), m_Name),
+                                               wxPoint(100, 100),
+                                               m_ParentWindow ?
+                                                   m_ParentWindow->FromDIP(wxSize(650, 500)) :
+                                                   wxSize(650, 500));
```

Comment:
This can be written more simply as
```suggestion
                                               wxWindow::FromDIP(wxSize(650, 500), m_ParentWindow));
```